### PR TITLE
#8 need to be clear, we must need to specify where we need to create …

### PR DESCRIPTION
**Pull request by 100daysofdevops**
…the temporary pod

### CODE REVIEW

File: exercises/01-creating-inspecting-pod/instructions.md [hunk 1]
```diff
@@ -11,6 +11,6 @@ In this exercise, you will practice the creation of a new Pod in a namespace. On
 5. List the Pod and ensure that the container is running.
 6. Log into the container and run the `ls` command. Write down the output. Log out of the container.
 7. Retrieve the IP address of the Pod `mypod`.
-8. Run a temporary Pod using the image `busybox`, shell into it and run a `wget` command against the `nginx` Pod using port 80.
+8. Run a temporary Pod in ckad-prep namespace using the image `busybox`, shell into it and run a `wget` command against the `nginx` Pod using port 80.
```
Comment (bmuschko) [inline exercises/01-creating-inspecting-pod/instructions.md:14]: Can you change this to "in the `ckad-namespace`"?

**Review by bmuschko — CHANGES_REQUESTED** [review]

File: exercises/01-creating-inspecting-pod/instructions.md (hunk 1 above)
Comment (100daysofdevops) [reply exercises/01-creating-inspecting-pod/instructions.md:14]: Yes, that is the intention of this PR; even in the solution, you created the POD in ckad-prep namespace :-)

Comment (bmuschko) [reply exercises/01-creating-inspecting-pod/instructions.md:14]: What I was getting at here is missing article "the". Can you please correct that?

**Comment by bmuschko** [comment]
Thanks for the pull request but I won't merge as-is as my requested change hasn't been incorporated. I made the change myself: https://github.com/bmuschko/ckad-crash-course/commit/c9f33014eea72400ce70edeb889d86bc16fe8759

**Pull request closed without merging**